content/css-blog.md

```diff
@@ -0,0 +1,36 @@
+---
+title: When CSS seems scary
+image: /images/writing.jpg
+imageMeta:
+  attribution:
+  attributionLink:
+featured: true
+authors:
+  - anne-greeth-schot-vanherwijnen
+date: Wed Oct 6 2021
+tags:
+  - css
+  - tech
+  - beginner
+---
+
+In this blogpost I want to show you how I tackle CSS. There are so many CSS rules, so many properties and options that it might feel overwhelming. I would recommend you to start small. 
+
+If we take the example of a HTML button element. You might like it the way it is or you might prefer to give it a bit more charm.  
+
+# Tip 1: Open the browser
+Often when I want to (re)design something I open up the dev-tools in the browser by right clicking “inspect element” and I’ll just write my CSS right there. The browser will give you auto-completion and a ton of options and you can see your changes directly.
+
+# Tip 2: Step by step
+Don’t think you have to know right away where your design will go. You can just start bit by bit changing what is already there. I usually start with some spacing, the height/width and color.
+
+# Tip 3: Steal like an artist
+Luckily a lot of people with CSS experience have built awesome codepens and libraries. You can steal from those by just copy-pasting or steal from them by understanding and adapting what they made. Often when you try to describe what you are looking for you will either find some code that does exactly that, learn better the different building block of what you want to built (and can probably start building yourself) or know what your next google search should be. 
+
+# Tip 4: Kill the perfectionist
+The difficult thing with design is that we know our taste and we know what others are capable of. This might increase the bar to start writing your CSS, because somebody else might know a better or more beautiful solution. However, they also had to learn how to get there. If your CSS does what it needs to and looks the way you like (at this point in time), you’ve done it! You wrote CSS!
```

Comment:
What do you mean by `might increase the bar to start writing your CSS`? How about "put up a hurdle for writing your CSS"? is that what you mean?